django/contrib/admin/static/admin/js/vendor/jquery/jquery.js

```diff
@@ -1077,7 +1077,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 
 	// Support: IE<10
 	// Check if getElementById returns elements by name
-	// The broken getElementById methods don't pick up programatically-set names,
```

Comment:
yeah this is regularly imported from jquery each release so will get overwritten